hyundai_kia_connect_api/Vehicle.py

```diff
@@ -45,7 +57,15 @@ class DayTripInfo:
     """Day Trip Info"""
 
     yyyymmdd: str = None
-    summary: TripInfo = None
```

Comment:
Be removing this it will break other regions.   I also think adding the below will break other regions.  Can we conform to existing?